docs/design/features/StringDeduplication.md

```diff
@@ -0,0 +1,196 @@
+# **String Deduplication**
```

Comment:
Nit: The extra "**" around the chapter titles are not necessary. Hashes are enough to tag chapter titles in markdown.

Reply:
changed. I just used the Typora..that's what it created.